.github/workflows/go.yml

```diff
@@ -0,0 +1,79 @@
+name: Go
+
+on: 
+  push:
+    branches:
+      - master
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+  pull_request:
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        go-version:
+          - 1.15
+          - 1.16
+        postgresql-image:
+          - postgres:9.4
+          - postgres:9.5
+          - postgres:9.6
```

Comment:
Do we support these older versions?
Can we use more recent versions instead of 9.4 & 9.5?

Reply:
Good catch. Updated according to https://www.percona.com/services/policies/percona-software-support-lifecycle